src/BatteryStateOfChargeService.h

```diff
@@ -17,6 +18,11 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
 private:
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
+    QTime myTime;
+    QTime *prevTime;
+    double myVoltage;
```

Comment:
I don't believe storing voltage is necessary in this class, so this private member is not needed.